apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -0,0 +1,129 @@
+
+For this example, we will present the following possible templates in Headless CMS for the product:
+
+- [Slug](#rendering-within-a-slug): `/apple-magic-mouse/p`
+- [Department](#rendering-within-a-department): `/technology/`
+- [Product Page generic template](#rendering-the-generic-product-page-template).
+
+#### Rendering within a slug
+
+When using PDPs, this type of page template is recognized by the slug pattern. In the example, we will use the [Apple Magic Mouse product page](https://starter.vtex.app/apple-magic-mouse/p) which has the slug `/apple-magic-mouse/p`.
+
+![PDP Slug template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpslug___95ada5a5b24e23e34c04f4837d11ade7.png)
+
+Adding the slug to this template makes it specific, prompting FastStore to recognize and prioritize it for rendering on the product page for the customer.
+However, for certain cases, you might want to use a specific template for products within a `subcategory` or `department` instead of relying on the slug. In such instances, the template can behave differently based on input values beyond the slug: `subcategory`, `category`, `department`, or the generic Product Page template.
+
+#### Rendering within a `department`
+
+Products in the VTEX catalog fall under a `category` tree, typically `/department/category/subcategory/`. For example, the product mentioned is in the `Technology department`, allowing you to create a `department` template.
+
+![PDP Department template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpdepartment___387cf641f675d05666afeafc65a3e56b.png)
+
+If this product is under a `subcategory`, you can create a corresponding template. The category tree is used in the template input, separated by slashes.
+
+![PDP subcategory template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpsubcategory___566be1fd3bba2db777dd18881933292c.png)
+
+You can also see the category hierarchy through the breadcrumb.
+
+![PDP breadcrumb](https://vtexhelp.vtexassets.com/assets/docs/src/breadcrumb___c9c12da51f685572dfb3a9403168b26a.png)
+
+#### Rendering the generic Product Page template
+If no matches are found for the slug or `department`, the generic template will be used to render the PDP. In this case, create a Headless CMS document with the `Product Page` option, and in the **Section** tab, leave the template field empty.
+
+![PDP general template](https://vtexhelp.vtexassets.com/assets/docs/src/pdpgeneral___cdeeaf6060b529aea029e2e5814efc50.png)
+
+Possible templates in CMS from this product:
+
+- Slug: `/apple-magic-mouse/p`
+- Department: `/technology/`
+- PDP generic template: (empty value)
+
+## Product List Page (PLP) Template
+
+The PLP multiple-page template approach follows a logic similar to the Product Page template. However, PLPs FastStore relies only on the slug information to find and render the appropriate page template. 
+
+The order to find and render is as follows:
+
+1.  A PLP template matches the slug.
```

Comment:
The indentation is a little bit off
<img width="909" alt="image" src="https://github.com/vtex/faststore/assets/3356699/57dd3f61-d7a0-45fc-b40e-cc376db617ff">

Reply:
nice catch. done [here](https://github.com/vtex/faststore/pull/2236/commits/aeb41901d0f6cc792c01ce4c6cd63eca2d102f18).